contrib/drivers/mysql/mysql_z_unit_model_test.go

```diff
@@ -4786,3 +4787,22 @@ func Test_Model_FixGdbJoin(t *testing.T) {
 		t.Assert(gtest.DataContent(`fix_gdb_join_expect.sql`), sqlSlice[len(sqlSlice)-1])
 	})
 }
+
+func TestOrderByStatementGenerated(t *testing.T) {
+	gtest.C(t, func(t *gtest.T) {
+		array := gstr.SplitAndTrim(gtest.DataContent(`table_with_prefix.sql`), ";")
```

Comment:
Please use standalone sql for your uint testing case, do not be coupled with other unit testing cases.

Reply:
> Please use standalone sql for your uint testing case, do not be coupled with other unit testing cases.

OK